skabase/conftest.py

```diff
@@ -83,7 +83,7 @@ def setup_log_test_device():
     # run test device
     file_path = os.path.dirname(os.path.abspath(__file__))
     testdevice_path = os.path.abspath(os.path.join(file_path, os.curdir)) + "/SKATestDevice/SKATestDevice.py"
-    cmdline = 'python3 ' + testdevice_path + ' ' + '1 &'
+    cmdline = 'python3 ' + testdevice_path + ' ' + '01 &'
```

Comment:
Why this change?  Commit message just says what changed, but not why.

Reply:
The instance name of the device server which is getting registered in the script is 01. Locally we already had a device server named 1 but in the the pipeline, the execution failed. So we had to change it.
Strangely in earlier pipeline runs this error did not occur as this code was added long back.